### PR TITLE
cloud: customize doc content for TiDB Cloud in v7.1

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Frank945946 — APPROVED** [review]

File: sql-statements/sql-statement-load-data.md [hunk 1]
```diff
@@ -45,6 +37,8 @@ Fields ::=
 
 You can use `LOCAL` to specify data files on the client to be imported, where the file parameter must be the file system path on the client.
 
+If you are using TiDB Cloud, to use the `LOAD DATA` statement to load local data files, you need to add the `--local-infile` option to the connection string when you connect to TiDB Cloud. For example, `mysql --connect-timeout 15 -u '<user_name>' -h <host_name> -P 4000 -D test --ssl-mode=VERIFY_IDENTITY --ssl-ca=/etc/ssl/cert.pem -p<your_password> --local-infile`.
```
Comment (qiancai) [inline sql-statements/sql-statement-load-data.md:40]:
```suggestion
If you are using TiDB Cloud, to use the `LOAD DATA` statement to load local data files, you need to add the `--local-infile` option to the connection string when you connect to TiDB Cloud. For example,  use `mysql --connect-timeout 15 -u '<user_name>' -h <host_name> -P 4000 -D test --ssl-mode=VERIFY_IDENTITY --ssl-ca=/etc/ssl/cert.pem -p<your_password> --local-infile` to connect to TiDB Serverless, and use `mysql --connect-timeout 15 --ssl-mode=VERIFY_IDENTITY --ssl-ca=<CA_path> --tls-version="TLSv1.2" -u root -h <host_name> -P 4000 -D test -p<your_password> --local-infile` to connect to TiDB Dedicated.
```

File: sql-statements/sql-statement-load-data.md [hunk 2]
```diff
@@ -45,6 +37,8 @@ Fields ::=
...
+
```
[the remark this answers is not in the document]
Comment (qiancai) [reply sql-statements/sql-statement-load-data.md:41]: updated in https://github.com/pingcap/docs/pull/14349/commits/9673c14a1f633a7b4bbce34c6dd376d6969ebd59

File: sql-statements/sql-statement-lock-tables-and-unlock-tables.md [hunk 3]
```diff
@@ -17,9 +17,12 @@ TiDB enables client sessions to acquire table locks for the purpose of cooperati
 
 A table lock protects against reads or writes by other sessions. A session that holds a `WRITE` lock can perform table-level operations such as `DROP TABLE` or `TRUNCATE TABLE`.
 
-> **Warning：**
+> **Note：**
 >
-> To enable table locks, you need to set [`enable-table-lock`](/tidb-configuration-file.md#enable-table-lock-new-in-v400) to `true` in the configuration files of all TiDB instances.
+> The table locks feature is disabled by default.
+>
+> - For TiDB Self-Hosted, to enable the table locks feature, you need to set [`enable-table-lock`](/tidb-configuration-file.md#enable-table-lock-new-in-v400) to `true` in the configuration files of all TiDB instances.
```
Comment (qiancai) [inline sql-statements/sql-statement-lock-tables-and-unlock-tables.md:24]:
```suggestion
> - For TiDB Self-Hosted, to enable the table locks feature, you need to set [`enable-table-lock`](https://docs.pingcap.com/tidb/stable/tidb-configuration-file#enable-table-lock-new-in-v400) to `true` in the configuration files of all TiDB instances.
```

**Comment by easonn7** [comment]
LGTM

File: sql-statements/sql-statement-load-data.md [hunk 4]
```diff
@@ -45,6 +37,8 @@ Fields ::=
 
 You can use `LOCAL` to specify data files on the client to be imported, where the file parameter must be the file system path on the client.
 
+If you are using TiDB Cloud, to use the `LOAD DATA` statement to load local data files, you need to add the `--local-infile` option to the connection string when you connect to TiDB Cloud. For example,  use `mysql --connect-timeout 15 -u '<user_name>' -h <host_name> -P 4000 -D test --ssl-mode=VERIFY_IDENTITY --ssl-ca=/etc/ssl/cert.pem -p<your_password> --local-infile` to connect to TiDB Serverless, or use `mysql --connect-timeout 15 --ssl-mode=VERIFY_IDENTITY --ssl-ca=<CA_path> --tls-version="TLSv1.2" -u root -h <host_name> -P 4000 -D test -p<your_password> --local-infile` to connect to TiDB Dedicated.
```
Comment (hfxsd) [inline sql-statements/sql-statement-load-data.md:40]:
```suggestion
If you are using TiDB Cloud, to use the `LOAD DATA` statement to load local data files, you need to add the `--local-infile` option to the connection string when you connect to TiDB Cloud. For example, use the following command to connect to a TiDB Serverless cluster:

```
mysql --connect-timeout 15 -u '<user_name>' -h <host_name> -P 4000 -D test --ssl-mode=VERIFY_IDENTITY --ssl-ca=/etc/ssl/cert.pem -p<your_password> --local-infile
```

Use use the following command to connect to TiDB Dedicated cluster:

```
mysql --connect-timeout 15 --ssl-mode=VERIFY_IDENTITY --ssl-ca=<CA_path> --tls-version="TLSv1.2" -u root -h <host_name> -P 4000 -D test -p<your_password> --local-infile
```
```

File: sql-statements/sql-statement-load-data.md [hunk 5]
```diff
@@ -45,6 +37,20 @@ Fields ::=
 
 You can use `LOCAL` to specify data files on the client to be imported, where the file parameter must be the file system path on the client.
 
+If you are using TiDB Cloud, to use the `LOAD DATA` statement to load local data files, you need to add the `--local-infile` option to the connection string when you connect to TiDB Cloud. 
+
+- The following is an example connection string for TiDB Serverless:
```
Comment (hfxsd) [inline sql-statements/sql-statement-load-data.md:42]: @okJiang PTAL

Comment (okJiang) [reply sql-statements/sql-statement-load-data.md:42]: lgtm

**Review by hfxsd — APPROVED** [review]

**Pull request merged**